aibolit/patterns/method_siblings/method_siblings.py

```diff
@@ -19,26 +19,30 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-from typing import List
-from aibolit.types_decl import LineNumber
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
-
-import javalang
+from aibolit.ast_framework.ast_node import ASTNode
+from typing import List
 import re
 
 
 class MethodSiblings:
-    def __init__(self):
-        pass
-
-    def value(self, filename: str) -> List[LineNumber]:
+    '''
+    Count those methods, which
+    start with the same names
+    '''
+    def __compare_next_nodes(self, node: ASTNode, new_node: ASTNode) -> List[int]:
+        node_name = re.findall(r'[A-Z]?[a-z]+|[A-Z]+(?=[A-Z]|$)', node.name)
+        if node.node_index < new_node.node_index and len(node_name) > 1 and \
```

Comment:
We better move this compassion `node.node_index < new_node.node_index` to `value` method.

Reply:
Agree with you